src/hooks/useTxNotifications.ts

```diff
@@ -79,9 +76,11 @@ const useTxNotifications = (): void => {
         const txId = 'txId' in detail ? detail.txId : undefined
         const txHash = 'txHash' in detail ? detail.txHash : undefined
         const groupKey = 'groupKey' in detail && detail.groupKey ? detail.groupKey : txId || ''
+        const humanDescription = 'humanDescription' in detail ? detail.humanDescription : ''
 
         dispatch(
           showNotification({
+            title: humanDescription,
```

Comment:
What will it look like w/o a title?

Reply:
I went through each tx event that we show a notification for and made sure to add a humanDescription there, either from the CGW response or an appropriate fallback i.e. `Transaction #<nonce>` or `Module transaction`